client-v2/src/components/FrontsCAPIInterface/TagInput.js

```diff
@@ -51,57 +82,62 @@ const CAPITagInput = <T>({ onChange, placeholder }: CAPITagInputProps<T>) => (
       clearSelection
     }) => (
       <div>
-        <TagWrapper>
-          <TextInput
+        <SearchContainer>
+          <SearchTitle>{capitalize(searchType)}:</SearchTitle>
+          <SearchInput
             {...getInputProps({
               placeholder,
               onClear: clearSelection,
-              width: '100%'
+              width: '100%',
+              value: tagsSearchTerm,
+              onChange: input => {
+                onSearchChange(input, searchType);
+              }
             })}
           />
-          <TagDropdown>
-            <TagQuery params={{ q: inputValue }}>
-              {({ value }) => {
-                if (!value || !isOpen) {
-                  return false;
-                }
+        </SearchContainer>
+        <TagDropdown>
+          <TagQuery tagType={searchType} params={{ q: inputValue }}>
+            {({ value }) => {
+              if (!value || !isOpen) {
+                return false;
+              }
 
-                /**
-                 * Filter tags based on ids
-                 * think this is only for the test/test tag but sometime tags
-                 * come
-                 */
-                const seenIds = [];
+              /**
+               * Filter tags based on ids
+               * think this is only for the test/test tag but sometime tags
+               * come
```

Comment:
I wrote this and it meant to say "come through more than once" - would you mind adding it on to the end of the comment!? Sorry!

Reply:
Thanks!